hibernate-core/src/main/java/org/hibernate/sql/results/spi/ListResultsConsumer.java

```diff
@@ -33,8 +33,8 @@
 public class ListResultsConsumer<R> implements ResultsConsumer<List<R>, R> {
 
 	/**
-	 * Let's be reasonable, a row estimate greater than 1M rows is probably either a mis-estimation or bug,
-	 * so let's set 2^20 which is a bit above 1M as maximum collection size.
+	 * Let's be reasonable, a row estimated greater than 1M rows is probably either a mis-estimation or bug,
+	 * so let's set {@code 2^20} which is a bit above 1M as maximum collection size.
```

Comment:
```suggestion
	 * Let's be reasonable: a row estimate greater than 1M rows is probably either a misestimate or a bug,
	 * so let's set {@code 2^20} which is a bit above 1M as maximum collection size.
```

"row estimate" looks more correct to me.